src/actions/lendingActions.js

```diff
@@ -150,47 +147,42 @@ export const calculateLendingDepositTransactionEstimateAction = (
       asset,
     );
 
-    const estimateTransactions = aaveDepositNeededTransactions.map(({
-      to: recipient,
-      amount: value,
-      data,
-    }) => ({ recipient, value, data }));
-
-    const estimateTransaction: AccountTransaction = {
-      ...estimateTransactions[0],
-      sequentialTransactions: estimateTransactions.slice(1), // exclude first, take rest if exist
-    };
-
-    const estimate = await smartWalletService
-      .estimateAccountTransaction(estimateTransaction)
-      .catch(() => null);
-
-    dispatch({ type: SET_LENDING_DEPOSIT_TRANSACTION_ESTIMATE, payload: estimate });
+    const sequentialTransactions = aaveDepositNeededTransactions
+      .slice(1) // exclude first, take rest if exist
```

Comment:
I guess the only concern here can be is that if `aaveDepositNeededTransactions` can be undefined which will cause `.slice(1)` to fail, however, it's always an array so `.slice(1)` will always work. On single size array it will exclude first and become empty which means `sequentialTransactions` is empty, otherwise it will return anything that goes after 1 index. Does it makes sense?

Reply:
As long as we can be sure that it's always an array - we're all good!